config/main.py

```diff
@@ -73,45 +74,6 @@ def log_error(msg):
     syslog.closelog()
```

Comment:
Code duplicated with common.py

Reply:
Good catch. Missed to remove. Will  put an update.